changelog.txt

```diff
@@ -1,11 +1,37 @@
+4.25.0
+==========
+### New Features: 
+**MSAL.NET now advertises PKAuth support only on supported platforms**. See issues [#1849](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1849), [2302](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2302) for details.  
+**`WithPreferredAzureRegion` allows specifying an option to fallback to global endpoint if the region lookup fails**. See issue [#2287](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2287) for details.  
```

Comment:
we have an "Experimental Features" section now, I would move the regional items there, and I guess WAM as well.

Reply:
Resolved.